src/nxdoc/web-ui/web-ui-release-notes/web-ui-11-1-release-notes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Nuxeo Web UI 11.1 Release Notes
+description: Discover changes brought in our recent Nuxeo Web UI updates.
+review:
+    comment: ''
+    date: '2020-06-24'
+    status: ok
+toc: true
+labels:
+tree_item_index: 999
+---
+
+{{! multiexcerpt name='release-notes'}}
+
+{{! multiexcerpt name='matching-notes'}}
+{{#> callout type='info' heading='Upgrade Notes'}}
+This page mentions what's new. Refer to the [upgrade notes]({{page page='web-ui-11-1-upgrade-notes'}}) to transition to this version.
+{{/callout}}
+{{! /multiexcerpt}}
+
+## What's New in Web UI 11.1
+
+### Upload Experience for Large Files
+
+Web UI provides a safe experience when uploading files, even for large files in the 75gb range and when the upload goes on for hours due to a slow connection.
+
+If a network problem happens, upload is resumed whenever possible. Otherwise, guidance is provided so that you can decide what to do with the situation: e.g. create the document anyway and upload the attachment later.
+
+- [<a href="https://jira.nuxeo.com/browse/NXP-29189" target="_blank">NXP-29189</a>]
+- [<a href="https://jira.nuxeo.com/browse/NXP-28495" target="_blank">NXP-28495</a>]
+- [<a href="https://jira.nuxeo.com/browse/NXP-27346" target="_blank">NXP-27346</a>]
+- [<a href="https://jira.nuxeo.com/browse/NXP-27162" target="_blank">NXP-27162</a>]
```

Comment:
Another related ticket, where most of the Web UI improvements were made: https://jira.nuxeo.com/browse/NXP-28291. There are a bunch of related tickets for backend as well that are required by Web UI, sitting on our current sprint/

Reply:
Didn't put that one because it's already in the epic that is referenced first (NXP-29189) ;-) I planned to put that epic only at first, then noticed it's not the first time changes were made regarding upload so I referenced these additional web ui tickets too.